memgpt/agent_store/db.py

```diff
@@ -379,29 +380,33 @@ def list_data_sources(self):
             unique_data_sources = session.query(self.db_model.data_source).filter(*self.filters).distinct().all()
         return unique_data_sources
 
-    def query_date(self, start_date, end_date, offset=0, limit=None):
+    def query_date(self, start_date, end_date, limit=None, offset=0):
```

Comment:
small question - is there any reason why this line is a diff? (args got swapped?)

Reply:
This fixes what I reference in 2.1 above.  The order of arguments is different from that of the callers, unless I'm interpreting the intent of what is supposed to be returned. Maybe a previous version used the keywords and order didn't matter, but the current callers of these functions don't so order does matter.